cache/test_cacher.go

```diff
@@ -0,0 +1,138 @@
+// (c) 2019-2020, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package cache
```

Comment:
nit: could we move this into an `*_test.go` file? codecov seems to be flagging all of the fatal lines (which aren't triggered since it passes) as lines that aren't covered by the tests

Reply:
If we put in `*_test.go`, we won't be able to import it in `cache/metercacher`.